src/components/ActionMenu.vue

```diff
@@ -40,15 +61,44 @@
             </div>
             <div
               class="action-menu__option action-menu__option--top-bordered"
+              v-if="isStepSupported('filter')"
               @click="openStep('filter')"
             >
               Filter values
             </div>
-            <div class="action-menu__option" @click="openStep('fillna')">Fill null values</div>
-            <div class="action-menu__option" @click="openStep('replace')">Replace values</div>
-            <div class="action-menu__option" @click="openStep('sort')">Sort values</div>
-            <div class="action-menu__option" @click="createUniqueGroupsStep">Get unique values</div>
-            <div class="action-menu__option" @click="openStep('statistics')">
+            <div
+              class="action-menu__option"
+              v-if="isStepSupported('fillna')"
```

Comment:
@davinov , pas l'object de cette PR, mais on aurait pas moyen de factorise un peu la liste ? Si on garde la même ui, on a juste un label et une action pour chaque boutons.

Reply:
Yes tu as raison, ce serait bien de passer plutôt une liste et de faire un v-for dessus, comme on fait avec les boutons de la toolbar :+1: